bundles/org.eclipse.mita.base/src/org/eclipse/mita/base/types/validation/IValidationIssueAcceptor.java

```diff
@@ -27,21 +28,57 @@
  */
 public interface IValidationIssueAcceptor {
 
+	@EqualsHashCode
 	public static class ValidationIssue {
-		public static enum Severity {
-			ERROR, WARNING, INFO
-		}
 
 		private Severity severity;
 		private String message;
 		private String issueCode;
-		private EObject target;
+		private transient EObject target;
 		private EStructuralFeature feature;
-
+		
+		private static boolean shouldTest() {
+			return true;
+		}
+		private static String getTestString() {
+			return "Couldn't resolve reference to Type ''";
+		}
+		private static boolean testFun(String s) {
+			return s.startsWith(getTestString());
+		}
+		private static void singleBreakpoint() {
```

Comment:
debug leftover?

Reply:
This one is really handy in future debugging, I would leave it in